.codeclimate.yml

```diff
@@ -1,3 +1,8 @@
+checks:
```

Comment:
Change implying project setup should have their own PR to avoid missunderstanding.

Reply:
True, I have removed it and put it in another PR